src/components/addSourceWizard/hardcodedComponents/azure/costManagement.js

```diff
@@ -202,10 +210,258 @@ export const ExportSchedule = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {intl.formatMessage({
             id: 'cost.azure.createdAccountName',
-            defaultMessage: 'Storage account name created earlier',
+            defaultMessage: 'Created storage account name or existing storage account name',
+          })}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          <Text component={TextVariants.b}>
+            {intl.formatMessage({
+              id: 'cost.azure.resourceGroupName',
+              defaultMessage: 'Resource group name',
+            })}
+          </Text>
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {intl.formatMessage({
+            id: 'cost.azure.createdResourceGroupName',
+            defaultMessage: 'Resource group for the storage account',
           })}
         </TextListItem>
       </TextList>
+      <br />
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'cost.azure.exportNameFollowup',
+          defaultMessage: 'After configuring the daily export, enter the following:',
+        })}
+      </Text>
     </TextContent>
   );
 };
+
+const CostExportSelect = ({ handleSelect }) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState('Subscription');
+  const onToggle = (isOpen) => {
+    setOpen(isOpen);
+  };
+
+  const clearSelection = () => {
+    setSelected(null);
+    setOpen(false);
+  };
+
+  const onSelect = (event, selection, isPlaceholder) => {
+    if (isPlaceholder) {
+      clearSelection();
+    } else {
+      setSelected(selection);
+      setOpen(false);
+      handleSelect(selection);
+    }
+  };
+
+  return (
+    <Select
+      variant={SelectVariant.single}
+      aria-label="Select Input"
+      onToggle={onToggle}
+      onSelect={onSelect}
+      selections={selected}
+      isOpen={isOpen}
+    >
+      <SelectOption key="subscription" value="Subscription" />
+      <SelectOption key="resourceGroup" value="Resource Group" />
+      <SelectOption key="billingAccount" value="Billing Account" />
+      <SelectOption key="enrollment" value="Enrollment Account" />
+      <SelectOption key="management" value="Management Group" />
+      <SelectOption key="billingProfile" value="Billing Profile" />
+      <SelectOption key="invoiceSection" value="Invoice Section" />
+    </Select>
+  );
+};
+
+CostExportSelect.propTypes = {
+  handleSelect: PropTypes.func.isRequired,
+};
+
+export default CostExportSelect;
```

Comment:
this default export does not seem to be needed, also it should be at the bottom of the file